model-optimizer/extensions/ops/elementwise.py

```diff
@@ -22,13 +22,19 @@ def override_data_type_of_constant(node: Node):
         # power value 2, or when replacing Neg operation with Mul with -1 as second input.
         in_node_0 = node.in_port(0).get_source().node
         in_node_1 = node.in_port(1).get_source().node
+
+        if in_node_0.op != 'Const' and in_node_1.op != 'Const':
```

Comment:
I am lowering comment priority to nit. Since there it not necessary 
In old code without my changes also `node.op` was used
https://github.com/openvinotoolkit/openvino/blob/master/model-optimizer/extensions/ops/elementwise.py#L25-L30